pkg/installer/install.go

```diff
@@ -33,11 +35,13 @@ func (m *manager) Install(ctx context.Context) error {
 			installConfig, image, err = m.generateInstallConfig(ctx)
 			return err
 		}),
-
+		steps.Action(func(ctx context.Context) error {
+			return m.determineOutboundType(m.oc, m.sub)
+		}),
```

Comment:
This has to happen before we generate the installation configuration since the `generateInstallConfig` function uses the outbound type to set the installConfig outboundType parameter.  Otherwise it'll nil out and fail? or default to something bad.